app/config/router-config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider) {
+  $urlRouterProvider.when('/', '/home');
```

Comment:
You'll want to pass an empty string to $urlRouterProvider so that way the user doesn't have to explicitly go to /#!/home etc.